File: 12_leap_year.c
//C program to find if entered year is leap year or not

# include <stdio.h>

int main(){
    int year;
    
    printf("Enter a year: ");
    scanf("%d", &year);
    
    if((year%4==0 && year%100!=0) || year%400==0){
        printf("\n%d is a leap year.",year);
    }
    else{
        printf("\n%d is not a leap year.",year);
    }
    
    return 0;
    
}

File: 12_leap_year.py
#python program to find if entered number is leap or not

#input the year
year=int(input("Enter any year: "))

#condition for leap year
if((year%4==0 and year%100!=0) or year%400==0):
  print("Entered year ", year," is a leap year")
else:
  print("Entered year ", year, " is not a leap year")
